match/lib/match/runner.rb

```diff
@@ -12,25 +12,43 @@ def run(params)
       params[:workspace] = GitHelper.clone(params[:git_url], params[:shallow_clone], skip_docs: params[:skip_docs], branch: params[:git_branch])
       spaceship = SpaceshipEnsure.new(params[:username]) unless params[:readonly]
 
+      if params[:app_identifier].kind_of?(Array)
+        app_identifiers = params[:app_identifier]
+      else
+        app_identifiers = params[:app_identifier].split(/\s*,\s*/).uniq
+      end
+
       # Verify the App ID (as we don't want 'match' to fail at a later point)
-      spaceship.bundle_identifier_exists(params) if spaceship
+      if spaceship
+        app_identifiers.each do |app_identifier|
+          params[:app_identifier] = app_identifier
```

Comment:
I don't really like the pattern of mutating the `params` `Configuration` object for every iteration over the `app_identifiers` that we do. If these `SpaceshipEnsure` methods don't actually need `Configuration` objects, can we just provide them hashes with the right keys instead?